fieldservice/views/fsm_order.xml

```diff
@@ -0,0 +1,491 @@
+<odoo>
+    <!-- Operations -->
+    <record id="fsm_order_form" model="ir.ui.view">
+        <field name="name">fsm.order.form</field>
+        <field name="model">fsm.order</field>
+        <field name="arch" type="xml">
+            <form string="Order">
+                <header>
+                    <button
+                        id="action_complete"
+                        name="action_complete"
+                        string="Complete"
+                        class="oe_highlight"
+                        type="object"
+                        groups="fieldservice.group_fsm_user"
+                        attrs="{'invisible': [('stage_id', 'in', (%(fieldservice.fsm_stage_completed)d, %(fieldservice.fsm_stage_cancelled)d))]}"
+                    />
+                    <field name="is_button" invisible="1" />
+                    <button
+                        id="action_cancel"
+                        name="action_cancel"
+                        string="Cancel Order"
+                        type="object"
+                        groups="fieldservice.group_fsm_dispatcher"
+                        attrs="{'invisible': [('stage_id', 'in', (%(fieldservice.fsm_stage_completed)d, %(fieldservice.fsm_stage_cancelled)d))]}"
+                    />
+                    <field
+                        name="stage_id"
+                        widget="statusbar"
+                        options="{'fold_field': 'fold'}"
+                        domain="[('stage_type', '=', 'order'),
+                                    ('team_ids', 'in', (team_id, False))]"
```

Comment:
```suggestion
                        domain="[('stage_type', '=', 'order'),
                                    '|', ('team_ids', 'in', team_id), ('team_ids', '=', False)]"
```
Fixes Bug: 
Create a new order stage that has no team (or fsm team group is not enabled)
New stage is in kanban view, but the new stage is not shown in the order form view